app/code/community/Bolt/Boltpay/sql/bolt_boltpay_setup/install-0.0.9.php

```diff
@@ -44,34 +44,49 @@
 $statusTable = $installer->getTable('sales/order_status');
 $statusStateTable = $installer->getTable('sales/order_status_state');
 
-// Insert statuses
-$installer->getConnection()->insertArray(
-    $statusTable,
-    array(
-        'status',
-        'label'
-    ),
-    array(
-        array('status' => 'deferred', 'label' => 'Deferred'),
-    )
-);
+try {
```

Comment:
instead of try-catch, can you select from database first?

Reply:
completed